docs/tutorials/constraints.rst

```diff
@@ -0,0 +1,255 @@
+:orphan:
+
+.. doctest-skip-all
+
+******************************
+Defining Observing Constraints
+******************************
+
+Contents
+========
+
+* :ref:`constraints-built_in_constraints`
+* :ref:`constraints-user_defined_constraints`
+
+.. _constraints-built_in_constraints:
+
+Built-In Constraints
```

Comment:
Maybe this should be something like "Using built-in constraints"?  Or split it into "Introduction to constraints" and "Built-in constraints"?  Right now it's rather imposing-looking because it sounds like you need to know immediately what constraints are and how they are used.